kivy/uix/image.py

```diff
@@ -138,18 +145,22 @@ def get_image_ratio(self):
         :class:`~kivy.properties.ColorProperty`.
     '''
 
-    allow_stretch = BooleanProperty(False)
+    allow_stretch = BooleanProperty(False, deprecated=True)
```

Comment:
`allow_stretch` has been deprecated, but is in use from Kivy widgets.
That will show a deprecation warning in the console. Better to migrate these widgets as well (even in a different PR, just after this one is merged, but before **2.2.0** release).

E.g.
https://github.com/kivy/kivy/blob/e2c8f0753337cd693ae5b34de0b3bd3a481d725c/kivy/data/style.kv#L75

(We also have a few examples which are using `allow_stretch `, but we can migrate them later, even after **2.2.0**)

Reply:
https://github.com/kivy/kivy/pull/8169#discussion_r1148395736